ADBench/plot_graphs.py

```diff
@@ -22,13 +22,21 @@
     print("WARNING: `--show` enabled. This script can produce a lot of graphs and you may not wish to display all of them.\n")
 
 # Script constants
+TIMES_SUBSTRING = "times"
```

Comment:
It can not be **times.txt** because the file name looks something like this: **gmm_d2_K5_correctness_Finite.txt**. But I think that **\_times\_** would be better. So, as for CORRECTNESS_SUBSTRING

Reply:
Done